client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice.jsx

```diff
@@ -169,11 +169,13 @@ class PendingGSuiteTosNotice extends React.PureComponent {
 				showDismiss={ false }
 				key="pending-gapps-tos-acceptance-domain"
 				text={ translate(
-					'%(exclamation)s To activate your email {{strong}}%(emails)s{{/strong}}, click "Finish Setup".',
-					'%(exclamation)s To activate your emails {{strong}}%(emails)s{{/strong}}, click "Finish Setup".',
+					'%(exclamation)s Finish the setup of {{strong}}%(emails)s{{/strong}} in order to activate it.',
+					'%(exclamation)s Finish the setup of {{strong}}%(emails)s{{/strong}} in order to activate them.',
```

Comment:
There is no need to invite the user to click the `Finish Setup` as this should be pretty obvious.

Reply:
As above, maybe remove "in order" to keep things simple.